tests/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('index', () => {
     const secret = process.env.AUTH0_SECRET;
     delete process.env.AUTH0_SECRET;
     expect(() => withApiAuthRequired(jest.fn())).toThrow('"secret" is required');
-    expect(() => withPageAuthRequired()).not.toThrow();
+    expect(() => withPageAuthRequired(jest.fn())).not.toThrow();
```

Comment:
This is testing the server side `withPageAuthRequired`, so shouldn't take a function as it's first argument.

Reply:
Changed in https://github.com/auth0/nextjs-auth0/pull/268/commits/2b4b4c6e8d9dc23051e22ba777b22aa9a7bc8cda.